src/devices/cpu/hcd62121/hcd62121.cpp

```diff
@@ -307,6 +346,9 @@ void hcd62121_cpu_device::device_start()
 {
 	m_program = &space(AS_PROGRAM);
 
+	m_timer = timer_alloc(FUNC(hcd62121_cpu_device::timer_tick), this);
+	m_timer->adjust(attotime::from_seconds(1), 0, attotime::from_seconds(1));
```

Comment:
Where does this one-second tick come from?  Does the CPU have a secondary clock input?  Is it an external 1-second tick input, or is it pre-scaled on the chip?  (Seiko Epson PDA CPUs have a secondary oscillator that’s usually connected to a 32.768kHz watch crystal that has configurable dividers for generating the clock tick signal.)

Reply:
There's a secondary crystal oscillator that can be connected to the CPU, but it seems to be unrelated, since it's optional: it's present on Picky Talk, but on CFX9850GB both ends are connected with a resistor.

The 1-second tick feature is present on both systems. The way I measured it was by sampling the TIME register value after various timer waits, and it was being incremented roughly every second, including the cutoff at 60. It isn't very clear to me yet how the CPU is configuring this timer.